generators/voxel_generator.cpp

```diff
@@ -31,6 +31,8 @@ void VoxelGenerator::_b_generate_block(Ref<VoxelBuffer> out_buffer, Vector3 orig
 
 void VoxelGenerator::_bind_methods() {
 	// Note: C++ inheriting classes don't need to re-bind these, because they are bindings that call the actual virtual methods
-
-	ClassDB::bind_method(D_METHOD("generate_block", "out_buffer", "origin_in_voxels", "lod"), &VoxelGenerator::_b_generate_block);
+	BIND_VMETHOD(MethodInfo("generate_block",
```

Comment:
Looks like this will make it impossible to call `generate_block` anymore from script?